Tavisca.Bootcamp.LanguageBasics.Exercise2/Program.cs

```diff
@@ -13,18 +13,81 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string[] postTimes, string[] showTimes, string expected)
-        {
+        private static void Test(string[] postTimes, string[] showTimes, string expected)
+        {
             var result = GetCurrentTime(postTimes, showTimes).Equals(expected) ? "PASS" : "FAIL";
             var postTimesCsv = string.Join(", ", postTimes);
             var showTimesCsv = string.Join(", ", showTimes);
-            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
+            Console.WriteLine($"[{postTimesCsv}], [{showTimesCsv}] => {result}");
         }
 
-        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
-        {
+        public static string GetCurrentTime(string[] exactPostTime, string[] showPostTime)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+            DateTime PostTime, CurrentTime, min = DateTime.Parse("00:00:00"), max = DateTime.Parse("23:59:59");
+            string ShowTime;
+
+            for(int i=0; i<exactPostTime.Length-1; i++)

```

Comment:
this loop verifies impossible occurrences . 
So can't we move this to a private method ?
That will make your function more  readable .